Cargo.toml

```diff
@@ -321,7 +321,7 @@ postgres-openssl = { version = "0.5.0", default-features = false, features = ["r
 pulsar = { version = "6.3.0", default-features = false, features = ["tokio-runtime", "auth-oauth2", "flate2", "lz4", "snap", "zstd"], optional = true }
 rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
 rand_distr = { version = "0.4.3", default-features = false }
-rdkafka = { version = "0.35.0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
+rdkafka = { version = "0.36.2", default-features = false, features = ["tokio", "libz", "ssl", "zstd", "sasl", "curl", "zstd-pkg-config"], optional = true }
```

Comment:
0.37 of rdkafka has breaking changes so 36.2 is the highest we can go to with simpler upgrades